src/SFA.DAS.ApprenticePortal.UnitTests/TestContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SFA.DAS.ApprenticePortal.UnitTests.Hooks;
+
+namespace SFA.DAS.ApprenticePortal.UnitTests
+{
+    public class TestContext 

```

Comment:
I quite like the pattern we use in APIM where we don't need the bindings - the bits of the context are lazily instantiated.

https://github.com/SkillsFundingAgency/das-apim-endpoints/blob/master/src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/TestContext.cs

I don't forsee the Portal having too many tests, so I don't think we need to change it now.  I'd like to use that pattern on new projects though.

Reply:
I'm happy to use the pattern in APIM, but I do think there is value in implementing the Feature here as the menu page has several logic statements (at the moment) and currently we are not testing these flows